docs/data/base/components/snackbar/snackbar.md

```diff
@@ -0,0 +1,133 @@
+---
+product: base
+title: Unstyled React Snackbar component and hook
+components: SnackbarUnstyled
+githubLabel: 'component: snackbar'
+---
+
+# Unstyled snackbar
+
+<p class="description">The SnackbarUnstyled component informs users that an action has been or will be performed by the app.</p>
+
+## Introduction
+
+A snackbar provides users with a brief message about app processes without interrupting their activity or experience.
+
+The `SnackbarUnstyled` component is built to appear temporarily in the corner of the screen to inform users of an action that the app is taking.
+After a set amount of time, it disappears on its own without requiring user interaction to dismiss it.
```

Comment:
It will only disappear if the duration is passed (autoHideDuration prop). By default, it will be not closed.

Reply:
Ah ok, I missed that. We should mention this prop in the doc since it's a big part of what defines a snackbar vs other kinds of popups and alerts. Maybe the **Basics** section would be a good place to point it out.

Reply:
I have removed this line for accuracy.